doc/api/vm.md

```diff
@@ -1330,6 +1330,7 @@ are not controllable through the timeout either.
 [`vm.createContext()`]: #vm_vm_createcontext_contextobject_options
 [`vm.runInContext()`]: #vm_vm_runincontext_code_contextifiedobject_options
 [`vm.runInThisContext()`]: #vm_vm_runinthiscontext_code_options
+[`vm.Script`]: #vm_class_vm_script
```

Comment:
not sure how to deal with this? There's no existing `vm.Module` entry here, so this is probably wrong 😬

Reply:
Links for argument types are auto-populated by tools/doc/type-parser.js, so this line should be removed and type-parser.js updated to handle `vm.Script`.

Reply:
Thanks! Gave that a try